src/apps/logging/logging.cpp

```diff
@@ -195,24 +195,19 @@ namespace ccfapp
       };
       // SNIPPET_END: log_record_prefix_cert
 
-      install_with_auto_schema<LoggingRecord::In, bool>(
-        Procs::LOG_RECORD, json_adapter(record), Write);
+      install(Procs::LOG_RECORD, json_adapter(record), Write)
+        .set_auto_schema<LoggingRecord::In, bool>();
       // SNIPPET: install_get
```

Comment:
Outside of this PR: I think we need to make this a `SNIPPET_START`, `SNIPPET_END` as it only renders the first line:
![image](https://user-images.githubusercontent.com/42961061/76873194-5b3e0c00-6865-11ea-9105-02265c182f8c.png)

Reply:
Added in this PR, since I'm touching it anyway.